src/Servers/IIS/AspNetCoreModuleV2/DefaultRules.ruleset

```diff
@@ -12,88 +12,96 @@
     <Rule Id="C26116" Action="Error" />
     <Rule Id="C26117" Action="Error" />
     <Rule Id="C26130" Action="Error" />
-    <Rule Id="C26135" Action="Error" />
+    <Rule Id="C26135" Action="None" /> <!-- Missing annotation <annotation> at function '<func>' -->
     <Rule Id="C26140" Action="Error" />
-    <Rule Id="C26160" Action="Error" />
+    <Rule Id="C26160" Action="None" /> <!-- Caller possibly failing to hold lock '<lock>' before calling function '<func>'. -->
     <Rule Id="C26165" Action="Error" />
     <Rule Id="C26166" Action="Error" />
     <Rule Id="C26167" Action="Error" />
-    <Rule Id="C26400" Action="Error" />
-    <Rule Id="C26401" Action="Error" />
+    <Rule Id="C26400" Action="None" /> <!-- Do not assign the result of an allocation or a function call with an owner<T> return value to a raw pointer -->
+    <Rule Id="C26401" Action="None" /> <!-- Do not delete a raw pointer that is not an owner<T> -->
     <Rule Id="C26402" Action="Error" />
     <Rule Id="C26403" Action="Error" />
     <Rule Id="C26404" Action="Error" />
     <Rule Id="C26405" Action="Error" />
     <Rule Id="C26406" Action="Error" />
     <Rule Id="C26407" Action="Error" />
     <Rule Id="C26408" Action="Error" />
-    <Rule Id="C26409" Action="Error" />
+    <Rule Id="C26409" Action="None" /> <!-- Avoid calling new and delete explicitly, use std::make_unique<T> instead -->
     <Rule Id="C26410" Action="Error" />
     <Rule Id="C26411" Action="Error" />
-    <Rule Id="C26414" Action="Error" />
-    <Rule Id="C26415" Action="Error" />
+    <Rule Id="C26414" Action="None" /> <!-- Move, copy, reassign or reset a local smart pointer. -->
+    <Rule Id="C26415" Action="None" /> <!-- Smart pointer parameter is used only to access contained pointer. Use T* or T& instead. -->
     <Rule Id="C26416" Action="Error" />
     <Rule Id="C26417" Action="Error" />
-    <Rule Id="C26418" Action="Error" />
+    <Rule Id="C26418" Action="None" /> <!-- Shared pointer parameter is not copied or moved. Use T* or T& instead. -->
     <Rule Id="C26426" Action="Error" />
     <Rule Id="C26427" Action="Error" />
-    <Rule Id="C26429" Action="None" />
-    <Rule Id="C26430" Action="Error" />
+    <Rule Id="C26429" Action="None" /> <!-- Symbol is never tested for nullness, it can be marked as gsl::not_null. -->
+    <Rule Id="C26430" Action="None" /> <!-- Symbol '<var>' is not tested for nullness on all paths  -->
     <Rule Id="C26431" Action="Error" />
-    <Rule Id="C26432" Action="None" />
-    <Rule Id="C26433" Action="Error" />
-    <Rule Id="C26434" Action="Error" />
+    <Rule Id="C26432" Action="None" /> <!-- If you define or delete any default operation in the type 'type-name', define or delete them all -->
+    <Rule Id="C26433" Action="None" /> <!-- Function should be marked with override -->
+    <Rule Id="C26434" Action="None" /> <!-- Function 'derived::function' hides a non-virtual function 'base::function' -->
```

Comment:
Because there are too many errors to handle in this single PR, so I disabled most rules that caused errors so we can split the fixes into separate PRs.